rocky/rocky/views/tasks.py

```diff
@@ -35,6 +36,7 @@ def get(self, request, *args, **kwargs):
 
 class TaskListView(OrganizationView, ListView):
     paginate_by = 20
+    paginator_class = RockyPaginator
```

Comment:
Is the variable paginator or paginator_class? The docs seems to point at paginator?
https://docs.djangoproject.com/en/5.0/topics/pagination/#using-paginator-in-a-view-function